lib/fastlane/plugin/revenuecat_internal/helper/versioning_helper.rb

```diff
@@ -153,8 +147,10 @@ def self.increase_version(current_version, type_of_bump, snapshot)
           :major
         elsif change_types.include?("feat")
           :minor
-        else
+        elsif (change_types & PUBLIC_CHANGE_LABELS).size > 0
```

Comment:
@tonidero this could use the map of all labels and their expected type you suggested in https://github.com/RevenueCat/fastlane-plugin-revenuecat_internal/pull/39#discussion_r995758518